models/descriptions/foundation-models/model-card-fill-mask-bert-base-cased.md

```diff
@@ -19,6 +19,9 @@ Token Classification|Token Classification|[Conll2003](https://huggingface.co/dat
 Question Answering|Extractive Q&A|[SQUAD (Wikipedia)](https://huggingface.co/datasets/squad)|[extractive-qa.ipynb](https://aka.ms/azureml-ft-sdk-extractive-qa)|[extractive-qa.sh](https://github.com/Azure/azureml-examples/blob/sitaram/finetunenotebooks/cli/foundation-models/system/finetune/question-answering/extractive-qa.sh)
 
 
+### Model Evaluation
 
-
+|Task|Use case|Dataset|Python sample (Notebook)|
+|---|--|--|--|
+|Fill Mask||[imdb](https://huggingface.co/datasets/imdb)|[evaluate-model-fill-mask.ipynb](https://aka.ms/azureml-eval-sdk-fill-mask/)|
```

Comment:
notebook pointing to URL is: https://github.com/Azure/azureml-model-evaluation/blob/main/sample_implementations/model_evaluation_dpv2/evaluation_of_base_models/model_evaluation_subgraph/evaluate_text_translation_base_models.ipynb 

which is for text-translation. Is this expected?

Reply:
Thanks for clarifying. Notebooks not present and link will be updated once notebook are added.